node_app/models/Immunization.js

```diff
@@ -0,0 +1,146 @@
+/*
+Immunization {
+  vaccine: string
+  manufacturer: string "Organization/id"
+  lotNumber: string
+  expiration: string "YYYY, YYYY-MM, or YYYY-MM-DD"
+  patient: string "Patient/id"
+  encounter: string "Encounter/id"
+  status: enum (completed, entered-in-error, not-done)
+  statusReason: enum (IMMUNE, MEDPREC, OSTOCK, PATOBJ) i.e. immunity, medical precaution, product out of stock, patient objection
+  occurrence: string "YYYY, YYYY-MM, YYYY-MM-DD or YYYY-MM-DDThh:mm:ss+zz:zz"
+  primarySource: boolean
+  location: string "Location/id"
+  site: enum (LA, RA) i.e. left arm, right arm
+  route: enum (IDINJ, IM, NASINHLC, IVINJ, PO, SQ, TRNSDERM) i.e. , inj intramuscular, inhalation nasal, inj intravenous, swallow, inj subcutaneous, transdermal
+  doseQuantity: decimal
+  doseUnit: string
+  performer: [string] "{Practitioner/PractitionerRole/Organization}/id"
```

Comment:
We should also capture the "note" field (this is what vaccine administrators will use to capture additional annotations)

Reply:
This is captured in the Observation resource, in the value field (valueString in FHIR)

Reply:
Are we using the Observation resource instead of this (which is part of the Immunization resource)?
![image](https://user-images.githubusercontent.com/61951196/103236310-33a6c480-490a-11eb-9d42-60964180d888.png)

Reply:
Done